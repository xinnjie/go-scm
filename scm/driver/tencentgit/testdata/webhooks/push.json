{
  "object_kind": "push",
  "operation_kind": "create",
  "action_kind": "client push",
  "before": "b96850262fabfa9a8a9d28fff9040621958379f9",
  "after": "e99db09904cfa385a129305e7660395f161950a",
  "ref": "refs/heads/master",
  "user_id": 11323,
  "user_name": "git_user1",
  "user_email": "git_user1@tencent.com",
  "project_id": 165245,
  "repository": {
    "name": "z-test",
    "url": "git@tencent.com:test/z-test.git",
    "description": "",
    "homepage": "https://tencent.com/test/z-test",
    "git_http_url": "http://tencent.com/test/z-test.git",
    "git_ssh_url": "git@tencent.com:test/z-test.git",
    "visibility_level": 0
  },
  "commits": [
    {
      "id": "b96850262fabfa9a8a9d28fff9040621958379f9",
      "message": "fix readme",
      "timestamp": "2015-03-13T09:52:25+0000",
      "url": "https://tencent.com/test/z-test/commit/b96850262fabfa9a8a9d28fff9040621958379f9",
      "author": {
        "name": "alex",
        "email": "alex@tencent.com"
      },
      "added": [],
      "modified": [
        "README.md"
      ],
      "removed": []
    },
    {
      "id": "1480a4610ca01dd10cb5bc30d8a1b1ea568c09a1",
      "message": "update readme",
      "timestamp": "2015-03-13T09:51:06+0000",
      "url": "https://tencent.com/test/z-test/commit/b96850262fabfa9a8a9d28fff9040621958379f9",
      "author": {
        "name": "Bob",
        "email": "Bob@tencent.com"
      },
      "added": [],
      "modified": [
        "README.md"
      ],
      "removed": []
    }
  ],
  "total_commits_count": 2
}
