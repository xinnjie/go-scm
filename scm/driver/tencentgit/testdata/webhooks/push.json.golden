{
    "Ref": "refs/heads/master",
    "After": "e99db09904cfa385a129305e7660395f161950a",
    "Repo": {
        "ID": "165245",
        "Namespace": "test",
        "Name": "z-test",
        "FullName": "test/z-test",
        "Perm": null,
        "Branch": "",
        "Private": false,
        "Clone": "http://tencent.com/test/z-test.git",
        "CloneSSH": "git@tencent.com:test/z-test.git",
        "Link": "https://tencent.com/test/z-test",
        "Created": "0001-01-01T00:00:00Z",
        "Updated": "0001-01-01T00:00:00Z"
    },
    "Commit": {
        "Sha": "1480a4610ca01dd10cb5bc30d8a1b1ea568c09a1",
        "Message": "update readme",
        "Author": {
            "Name": "Bob",
            "Email": "Bob@tencent.com",
            "Date": "0001-01-01T00:00:00Z",
            "Login": "Bob",
            "Avatar": ""
        },
        "Committer": {
            "Name": "git_user1",
            "Email": "git_user1@tencent.com",
            "Date": "0001-01-01T00:00:00Z",
            "Login": "git_user1",
            "Avatar": ""
        },
        "Link": "https://tencent.com/test/z-test/commit/b96850262fabfa9a8a9d28fff9040621958379f9"
    },
    "Sender": {
        "Login": "git_user1",
        "Name": "git_user1",
        "Email": "git_user1@tencent.com",
        "Avatar": ""
    }
}
